src/main/java/step02/InputView.java

```diff
@@ -0,0 +1,21 @@
+package step02;
+
+import java.util.Scanner;
+
+import static step02.LottoConfig.*;
+
+public class InputView {
+    private static Scanner scanner = new Scanner(System.in);
```

Comment:
초기화 및 재 할당 방지 차원에서 `final`도 추가되면 좋을 것 같습니다 😄  
```suggestion
    private static final Scanner SCANNER = new Scanner(System.in);
```